docpages/03_example_programs.md

```diff
@@ -629,23 +629,23 @@ int main()
 	});
 
 	bot.on_ready([&bot](const dpp::ready_t & event) {
-
-		dpp::slashcommand newcommand;
-		/* Create a new global command on ready event */
-		newcommand.set_name("blep")
-			.set_description("Send a random adorable animal photo")
-			.set_application_id(bot.me.id)
-			.add_option(
-				dpp::command_option(dpp::co_string, "animal", "The type of animal", true).
-					add_choice(dpp::command_option_choice("Dog", std::string("animal_dog"))).
-					add_choice(dpp::command_option_choice("Cat", std::string("animal_cat"))).
-					add_choice(dpp::command_option_choice("Penguin", std::string("animal_penguin")
-				)
-			)
-		);
-
-		/* Register the command */
-		bot.global_command_create(newcommand);
+	    if (dpp::run_once<struct register_bot_commands>()) {
+
+		    dpp::slashcommand newcommand;
+		    /* Create a new global command on ready event */
+		    newcommand.set_name("blep", "Send a random adorable animal photo", bot.me.id)
```

Comment:
this doesn't work.... the 3 parameter version is a constructor not the set_name method.